cross-media-measurement/src/test/kotlin/org/wfanet/measurement/common/identity/IdentifiersTest.kt

```diff
@@ -25,7 +25,7 @@ class IdentifiersTest {
 
   @Test
   fun `round trips`() {
-    for (i in listOf<Long>(0, 1, 10, 64, 1 shl 32, Long.MAX_VALUE)) {
+    for (i in listOf(0, 1, 10, 64, 1 shl 32, Long.MAX_VALUE)) {
       val externalId1 = ExternalId(i)
       val apiId1 = externalId1.apiId
       val externalId2 = ApiId(apiId1.value).externalId
```

Comment:
While you're cleaning up this file, can you import the Truth symbol for consistency with other tests?